website/content/api-docs/config.mdx

```diff
@@ -192,19 +186,17 @@ The table below shows this endpoint's support for
 
 The corresponding CLI command is [`consul config list`](/commands/config/list).
 
-### Parameters
+### Path Parameters
 
-- `dc` `(string: "")` - Specifies the datacenter to query. This will default to
-  the datacenter of the agent being queried. This is specified as part of the
-  URL as a query parameter.
+- `kind` `(string: <required>)` - Specifies the kind of the entry to list.
 
-- `kind` `(string: <required>)` - Specifies the kind of the entry to list. This
-  is specified as part of the URL.
+### Query Parameters
+
+- `dc` `(string: "")` - Specifies the datacenter to query. This will default to
```

Comment:
I agree with these changes. I'll make them if I can. I wasn't intending to change the text for the fields like this that I was only removing unnecessary words from, but since you've taken the time to review them, I'll see if I can fit these changes in.